src/contracts/libraries/GPv2Encoding.sol

```diff
@@ -38,6 +35,13 @@ library GPv2Encoding {
     /// @dev The stride of an encoded order.
     uint256 internal constant ORDER_STRIDE = 204;
 
+    /// @dev The order EIP-712 type hash.
+    ///
+    /// This type hash is computed as per the EIP-712 standard as the hash of
+    /// the `Order` struct fields: `keccak256("Order(...)")`.
+    bytes32 internal constant ORDER_TYPE_HASH =
```

Comment:
I'd be in favor of writing the hashed value explicitly in the code and (assuming the compiler doesn't precompute it anyway) replace it with the actual value immediately before deploying the contract.

Reply:
The line was just... so... long... But yes I agree, I even had it in at some point :stuck_out_tongue:

Reply:
> you don't use all fields. I would be explicit here and write the type struct into the comment else this might cause confusion

Yes, @fleupold suggested splitting the `Order` struct into two structs, one for the signed data and one for the extra encoded data (such as the digest, token indices, etc.) so that it is clearer what exactly is being signed.

Reply:
Boo :cry: 

> `TypeError: Only direct number constants and references to such constants are supported by inline assembly.`